python/cudf/cudf/pandas/fast_slow_proxy.py

```
@@ -881,12 +883,36 @@ def _assert_fast_slow_eq(left, right):
         assert_eq(left, right)
 
 
-class ProxyFallbackError(Exception):
+class FallbackError(Exception):
     """Raised when fallback occurs"""
 
     pass
 
 
+class OOMFallbackError(FallbackError):
+    """Warns when cuDF produces a MemoryError or an rmm.RMMError"""
```

All the docstrings should say "Raises" instead of Warns".